easybuild/easyblocks/n/nwchem.py

```diff
@@ -222,16 +227,21 @@ def configure_step(self):
                 raise EasyBuildError("Don't know how to set LIBMPI for %s", mpi_family)
             env.setvar('LIBMPI', libmpi)
 
-        if self.cfg['armci_network'] in ["OPENIB"]:
-            libmpi += " -libumad -libverbs -lpthread"
+        if not garoot:
+            if self.cfg['armci_network'] in ["OPENIB"]:
+                libmpi += " -libumad -libverbs -lpthread"
 
         # compiler optimization flags: set environment variables _and_ add them to list of make options
         self.setvar_env_makeopt('COPTIMIZE', os.getenv('CFLAGS'))
         self.setvar_env_makeopt('FOPTIMIZE', os.getenv('FFLAGS'))
 
         # BLAS and ScaLAPACK
-        self.setvar_env_makeopt('BLASOPT', '%s -L%s %s %s' % (os.getenv('LDFLAGS'), os.getenv('MPI_LIB_DIR'),
-                                                              os.getenv('LIBSCALAPACK_MT'), libmpi))
+        MPI_LIB_DIRS = ' '.join('-L' + d for d in os.getenv('MPI_LIB_DIR').split())
```

Comment:
@Micket Style-wise, this should be lower case since it's not a constant, but a variable.

```python
mpi_lib_dirs = ' '.join('-L' + d for d in os.getenv('MPI_LIB_DIR').split())